virttest/utils_net.py

```diff
@@ -1265,41 +1265,60 @@ def local_runner_status(cmd, timeout=None, shell=False):
     return process.run(cmd, verbose=False, timeout=timeout, shell=shell).exit_status
 
 
-def get_net_if(runner=None, state=None):
+def get_net_if(runner=None, state=None, session=None):
     """
     :param runner: command runner.
-    :param div_phy_virt: if set true, will return a tuple division real
-                         physical interface and virtual interface
+    :param state: regex to be used for state.
+    :param session: remote session object.
     :return: List of network interfaces.
     """
-    if runner is None:
-        runner = local_runner
+    cmd = "ip link"
     if state is None:
         state = ".*"
-    cmd = "ip link"
-    result = runner(cmd)
+    if session:
```

Comment:
@balamuruhans  it better write another function like get_remote_host_net_ifs(). I have a plan to rewrite get_net_ifs and get_sorted_net_ifs with netifaces module to make utils_net looks more clear.